libmesh

```diff
@@ -1 +1 @@
-Subproject commit 727d0685dc96e841c5805608653410ea0cd926a6
+Subproject commit 3287318f07a03b92c814508d614a7081cb52643e
```

Comment:
**Caution!** This contains a submodule update

Reply:
nargh!!

Reply:
can't we put `libmesh` into `.gitignore` so that it is only added with `git add -f` ?